src/components/Menu/Menu.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('Menu Component', () => {
   })
 
   test('renders skeleton correctly', async() => {
-    const { asFragment } = render(<Menu isLoading={true} />)
+    const { asFragment } = render(<Menu isLoading={true} items={[]} />)
```

Comment:
it could make sense that items were not required in case the menu was loading

Reply:
resolved